editorial_workflow/README.md

```diff
@@ -0,0 +1,25 @@
+# Editorial Workflow
+
+This is a copy of the Editorial workflow used by editors when reviewing staff
+or community blog posts to the [rOpenSci blog](https://ropensci.org/blog).
+
+This is a relatively simple workflow and a script is not even necessary. 
```

Comment:
this is all really nice :sparkles: in a tech note I'm sure there'll be aha moments for readers